src/appmixer/asana/projects/CreateProject/component.json

```diff
@@ -204,26 +31,36 @@
         {
             "name": "newProject",
             "options": [
```

Comment:
@sayam-nasir  you've removed some of the output properties e.g `archived`, `notes`, `workspace`.... and some more are gone. Is there a reason for that? if so, it's a breaking change then
Also, could you please update the labels to match our guiidelines (Title Case format)?

Reply:
@vtalas this has beem fixed, we dont need a change in this